homeassistant/components/tado/sensor.py

```diff
@@ -225,6 +226,9 @@ def _async_update_zone_data(self):
             self._state = self._tado_zone_data.open_window
```

Comment:
```suggestion
            self._state = bool(self._tado_zone_data.open_window or self._tado_zone_data.open_window_detected)
```
https://github.com/home-assistant/core/issues/30873#issuecomment-576190768
According the above I don't think you need a new sensor. I think the existing sensor should check either condition.  Please double check though.

Reply:
There is a problem with causality. First openWindowDetected will contain true/false, After clicking on the "open window mode" in the app, the "open_window" will contain a bigger object like this: `
detectedTime: 2020-05-05T17:51:56Z 
durationInSeconds: 900 
expiry: 2020-05-05T18:06:56Z 
remainingTimeInSeconds: 825
--
` and openWindowDetected will vanish. 

Meaning, if you only react on openWindowDetected from the tado api, you will *not* get the state attributes; this would mutate to a breaking change for people who were relying on those. IMHO it would be wrong to combine the two different responses into one. If you insist, i can consider merging together, but I think its wrong.

Reply:
Maybe I'm missing something but `self._state_attributes = self._tado_zone_data.open_window_attr` is always set regardless of the value of `self._tado_zone_data.open_window_attr` or `self._tado_zone_data.open_window_detected` so I'm not setting where they would not get the state attributes.